src/include/message.h

```diff
@@ -55,6 +55,26 @@ struct message
    void* data;        /**< The message data */
 } __attribute__ ((aligned (64)));
 
+/** @struct
+ * Defines a tuple
+ */
+struct tuple
+{
+   char** data;                   /**< The data */
+   struct tuple* next;            /**< The next tuple */
+} __attribute__ ((aligned (64)));
+
+/** @struct
+ * Defines a query
+ */
+struct query
+{
+   char names[MAX_NUMBER_OF_COLUMNS][MISC_LENGTH]; /**< The column names */
```

Comment:
I feel that we probably need to change the note here, judging from the code below this is more like a response to a query instead of the query itself? And is 8 big enough as the max column number?

Reply:
I think we are ok with 8 for now

Reply:
Do you want to reconsider the naming here? `query_response` makes more sense to me.

Reply:
Hmm I think that is fine, since we have the comment there. I just feel that this struct is more of the response to a query, judging from the usage in `pgmoneta_query_execute`, unless we have other usage later.

Reply:
Type it up, and lets take a look at it